kobo/apps/accounts/tests/test_api_tos.py

```diff
@@ -0,0 +1,26 @@
+from django.utils.timezone import now
+from django.urls import reverse
+from rest_framework import status
+from model_bakery import baker
+
+from kpi.tests.base_test_case import BaseTestCase
+
+
+class TOSTestCase(BaseTestCase):
+    def setUp(self) -> None:
+        self.url = reverse(self._get_endpoint('tos'))
+        self.user = baker.make(
+            'auth.User', username='spongebob', email='me@sponge.bob'
+        )
+        self.client.force_login(self.user)
+
+    def test_post(self):
+        # Prepare and send the request with empty payload
+        time = now().strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
This looks like an 8601 string. I wonder if there's something like an `.isoformat()` function in Python that's equivalent to this. 

- JS: [`.toDateString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)
- wikipedia: [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)

Reply:
(was talking to David and he says y'all considered this already)

- date formatting code like this is present elsewhere
    - it might make sense to use a constant or another function
    - but being that this api is short-term/internal, and the PR is pretty small, it makes sense to be consistent with it rather than departing from the norm or changing a bunch of unrelated code for uniformity